### PR TITLE
Run pdsm after main summary

### PR DESCRIPTION
**Pull request by robotblake**
The idea here is to run the tool that updates glue immediately after main summary finishes.

Note: I'm still planning on turning this into a proper operator soon, just fighting code at the moment. 😞

### CODE REVIEW

**Comment by jasonthomas** [comment]
I am r+, but @mreid-moz for final review.

File: dags/main_summary.py [hunk 1]
```diff
@@ -210,6 +210,22 @@
     uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/client_count_daily_view.sh",
     dag=dag)
 
+main_summary_glue = EMRSparkOperator(
+    task_id="main_summary_glue",
+    job_name="Main Summary Update Glue",
+    execution_timeout=timedelta(hours=2),
```
Comment (mreid-moz) [inline dags/main_summary.py]: Please add an `email` param to notify you if something goes wrong

**Comment by mreid-moz** [comment]
Also I'm not sure what's going on with the CI failure.

File: dags/main_summary.py [hunk 2]
```diff
@@ -210,6 +210,22 @@
     uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/client_count_daily_view.sh",
     dag=dag)
 
+main_summary_glue = EMRSparkOperator(
```
Comment (fbertsch) [inline dags/main_summary.py]: This probably doesn't need to be an EMR instance, saving us a bit of bootstrap time. No change needed, just a consideration for the future.

**Pull request merged**